### ISSUE TITLE
Any chance for the option to open a link inside chrome without default browser status?

### ISSUE THREAD

**Issue by atomic1fire** [flackr/circ#336]
https://developer.chrome.com/apps/browser

is going to probably be released to stable eventually.

**Comment by talons235** [comment]
I agree, making this app able to run within a chrome tab, or just temporarily add a server from the click of a link would make this amazing to use.

**Comment by atomic1fire** [comment]
I don't think you can run the app inside a tab, I was just asking if we could open a link from irc chat in chrome.

    person: goto http://www.example.com

Default behavior is to open a link inside the default browser. I was just asking for a way to ignore the default if we want to use chrome without setting it as default.

If that sounds counter intuitive I totally understand.
